### PR TITLE
Fix series stats merge

### PR DESCRIPTION
**Pull request by yeya24**
* [ ] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

## Changes

Fix a bug when merging `SeriesFetchedSizeSum`.

### CODE REVIEW

**Review by fpetkovski — DISMISSED** [review]

File: pkg/store/hintspb/custom_test.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+import (
+	"testing"
+
+	"github.com/efficientgo/core/testutil"
+)
+
+func TestQueryStatsMerge(t *testing.T) {
+	s := &QueryStats{
+		BlocksQueried:          1,
```
[the remark this answers is not in the document]
Comment (yeya24) [reply pkg/store/hintspb/custom_test.go:11]: Thanks for the suggestion. I am not sure how reflection can be used here. Any example you could share would be appreciated

[the remark this answers is not in the document]
Comment (fpetkovski) [reply pkg/store/hintspb/custom_test.go:14]: I was thinking we could use reflection to set all fields for `m` to 1, and all fields to `other` to 100. Then for assertions we would check that all fields for the result are 101.

[the remark this answers is not in the document]
Comment (fpetkovski) [reply pkg/store/hintspb/custom_test.go:14]: This way if we add a new field and don't change the merge, the test will fail.

[the remark this answers is not in the document]
Comment (yeya24) [reply pkg/store/hintspb/custom_test.go]: I have updated the test to use reflect. PTAL

**Review by fpetkovski — COMMENTED** [review]
Thanks, lgtm 👍

**Review by fpetkovski — APPROVED** [review]

**Pull request merged**